subprojects/docs/src/docs/userguide/build-cache/build_cache_debugging.adoc

```diff
@@ -196,91 +196,4 @@ Build cache key for task ':compileJava' is 8ebf682168823f662b9be34d27afdf77
 The log shows e.g. which source files constitute the `stableSources` for the `compileJava` task.
 To find the actual differences between two builds you need to resort to matching up and comparing those hashes yourself.
 
-Luckily, you do not have to capture this data yourself - the build scan plugin already takes care of this for you.
-Gradle Enterprise has the necessary data to diagnose the cache miss when comparing two build scans, and will show you the properties that caused a different cache key to be generated.
-
-[.screenshot]
-image::build-cache/adjacent-builds-input-comparison.png[]
-
-There are three ways to compare builds in Gradle Enterprise.
-
-1. Use the scan list to find the scans to compare, selecting them by clicking the respective image:build-cache/compare-small.svg[compare,role="inline-icon",width="30"] or image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon.
-2. From a scan that you wish to compare with another, click the image:build-cache/compare-small.svg[compare,role="inline-icon",width="30"] icon at the top of the scan, then select the second scan by clicking the  image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon from the list of scans.
-3. Click the image:build-cache/previous-next-scan.svg[workspace,role="inline-icon", width="22"] icon in the header of a build scan to show previous and subsequent build scans of the same workspace, then click the image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon next to the build you want to compare.
-
-[.screenshot]
-image::build-cache/adjacent-build-comparison.png[]
-
-It is also possible that task output caching for a cacheable task was disabled.
-When this happens the reason why caching was disabled for the task is reported on the info log level and in the build scan:
-
-[.screenshot]
-image::build-cache/caching-disabled.png[]
-
-[[diagnosing_cache_miss]]
-== Diagnosing the reasons for a cache miss
-
-Having the data from the last section at hand, you should be able to diagnose why the outputs of a certain task were not found in the build cache.
-Since you were expecting more tasks to be cached, you should be able to pinpoint a build which would have produced the artifact under question.
-
-Before diving into how to find out why one task has not been loaded from the cache we should first look into which task caused the cache misses.
-There is a cascade effect which causes dependent tasks to be executed if one of the tasks earlier in the build is not loaded from the cache and has different outputs.
-Therefore, you should locate the first cacheable task which was executed and continue investigating from there.
-This can be done from the timeline view in a build scan or from the task input comparison directly:
-
-[.screenshot]
-image::build-cache/first-non-cached-task.png[]
```

Comment:
> This can be done from the timeline view in a build scan

This is possible using scans.gradle.com.

> the task input comparison directly

This is not.